src/components/Comment/Comment.tsx

```diff
@@ -40,17 +40,19 @@ const commentControls = css`
   align-items: flex-start;
 `;
 
-const commentControlsButton = (pillar: Pillar) => css`
-  ${textSans.xsmall({ fontWeight: "bold" })}
-  margin-right: ${space[2]}px;
-  color: ${palette[pillar][400]};
-  background-color: transparent;
-  border: 0;
-  cursor: pointer;
-  :hover {
-    text-decoration: underline
+const commentControlsButtonWrapper = (pillar: Pillar) => css`
+  button {
+    ${textSans.xsmall({ fontWeight: "bold" })}
+    color: ${palette[pillar][400]};
+    background-color: transparent;
+    border: 0;
+    :hover {
+      text-decoration: underline
```

Comment:
You will probably also need to override `text-decoration-color`

Reply:
weirdly it seems to work without, however this led me down a rabbit whole in which i found that the links didnt have the same `text-decoration-color`

Reply:
I am going to apply `text-decoration-color` here nonetheless incase there is variation between browsers